src/test/java/uk/nhs/prm/deduction/e2e/queue/QueueMessageHelper.java

```diff
@@ -51,6 +54,27 @@ public SqsMessage getMessageContaining(String substring) {
         return found;
     }
 
+    public ArrayList<SqsMessage> getAllMessageContaining(String substring) {
+        log(String.format("Checking if message is present on : %s", this.queueUri));
+        ArrayList<SqsMessage> allMessages = new ArrayList<>();
```

Comment:
```suggestion
        List<SqsMessage> allMessages = new ArrayList<>();
```

Can we change this to the generic List interface please?